src/connector/src/parser/unified/upsert.rs

```diff
@@ -0,0 +1,109 @@
+use risingwave_common::types::DataType;
+
+use super::{Access, ChangeEvent, ChangeEventOperation};
+use crate::parser::unified::AccessError;
+
+/// `UpsertAccess` wraps a key-value message format into an upsert source.
+/// A key accessor and a value accessor are required.
+pub struct UpsertAccess<K, V> {
```

Comment:
`UpsertAccess<K, V>` -> `UpsertAccess<K: Access, V: Access>`. Add trait bound to make it clear.

Reply:
But we have a blank Default trait implementation for UpsertAccess.